src/components/ButtonGroupPicker/readme.md

```diff
@@ -0,0 +1,171 @@
+                    <ButtonOption label="Day" name="day" />
+                </ButtonGroupPicker>
+            </div>
+        );
+    }
+}
+
+<ButtonGroupPickerTry />;
+
+```
+
+##### ButtonGroupPicker multiple options selected
+
+```js
+import React from 'react';
+import { ButtonGroupPicker, ButtonOption } from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBold, faItalic, faUnderline, faStrikethrough } from '@fortawesome/free-solid-svg-icons';
+
+class ButtonGroupPickerTry extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { values: [
+            'bold',
+            'italic',
+        ]};
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    handleOnChange(values) {
+        this.setState({ values });
+    }
+
+    render() {
+        const { values } = this.state;
+        return (
+            <div className="rainbow-p-vertical_large rainbow-align-content_center rainbow-flex_wrap">
+                <ButtonGroupPicker 
+                    className="rainbow-m-around_medium" 
+                    value={values} 
+                    onChange={this.handleOnChange} 
+                    name="text-options-multiple"
+                    multiple 
+                >
+                    <ButtonOption label={<FontAwesomeIcon icon={faBold} />} name="bold" />
+                    <ButtonOption label={<FontAwesomeIcon icon={faItalic} />} name="italic" />
+                    <ButtonOption label={<FontAwesomeIcon icon={faUnderline} />} name="underline" />
+                    <ButtonOption label={<FontAwesomeIcon icon={faStrikethrough} />} name="strike" />
+                </ButtonGroupPicker>
+            </div>
+        );
+    }
+}
+
+<ButtonGroupPickerTry />;
+
+```
+
+##### ButtonGroupPicker with option disabled
+
+```js
+import React from 'react';
+import { ButtonGroupPicker, ButtonOption } from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAlignLeft, faAlignCenter, faAlignRight } from '@fortawesome/free-solid-svg-icons';
+
+class ButtonGroupPickerTry extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { values: []};
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    handleOnChange(values) {
+        this.setState({ values });
+    }
+
+    render() {
+        const { values } = this.state;
+        return (
+            <div className="rainbow-p-vertical_large rainbow-align-content_center rainbow-flex_wrap">
+                <ButtonGroupPicker 
+                    className="rainbow-m-around_medium" 
+                    value={values} 
+                    onChange={this.handleOnChange} 
+                    name="text-options-multiple"
+                >
+                    <ButtonOption label={<FontAwesomeIcon icon={faAlignLeft} />} name="left" disabled />
```

Comment:
disabled styles are not working as expected, for example, it changes icon color on hover, and if it is pressed it makes the animation, I am pretty sure this should change but check with @TahimiLeonBravo what should be the behavior